frontend/src/metabase/query_builder/components/AlertModals/AlertModals.jsx

```diff
@@ -381,7 +383,12 @@ export class DeleteAlertSection extends Component {
     const { onDeleteAlert } = this.props;
 
     return (
-      <DangerZone className="DangerZone mt4 pt4 mb2 p3 rounded bordered relative">
+      <DangerZone
+        className={cx(
+          AlertModalsS.AlertModalsContainer,
```

Comment:
I'm not sure why do we need an extra class here as it's not here before.

Reply:
Looks like this component is only used here.